docs/examples/cert-stream.rst

```diff
@@ -24,7 +24,7 @@ information.
 	# information.
 	#
 	# Author: achillean
-
+	from __future__ import print_function

```

Comment:
It looks like this is only necessary for Python versions older than 2.6? https://docs.python.org/2/library/__future__.html

I think it's ok if we only support Python 2.7.

Reply:
oh, ok. No problem, just omit the ``from __future__ ...` part. The print() function needs the parenthesis though if you want to run this copy & paste sample code under python3.